docs/spec.md

```diff
@@ -6317,33 +6317,47 @@ def baseline_type(x: Value | Placeholder | Type) -> Type:
 floating-point tensor types. This happens via converting quantized elements
 which represent integer values of the storage type into corresponding
 floating-point values of the expressed type using the zero point and scale
-associated with the quantized element type. At the moment, this function only
-works for per-tensor quantization. Per-axis quantization is work in progress
-([#1574](https://github.com/openxla/stablehlo/issues/1574)).
+associated with the quantized element type.
 
 ```python
+def get_zero_points(quantized_type, result_type):
+  if is_per_tensor_quantized(quantized_type):
+    return broadcast_in_dim(constant(zero_point(quantized_type), storage_type(quantized_type)), [], result_type)
+  if is_per_axis_quantized(quantized_type):
+    for i in index_space(result_type):
```

Comment:
I guess we need this loop for multi-axis quantized tensors?

Reply:
That is right. FWIW, [here](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#shape-computations ) we define `index_space` which helps us to use a linearized loop over the indices of tensor travered lexicographically.